InputTime/index.jsx

```diff
@@ -84,8 +86,8 @@ const InputTime = ({
       marginBottom: 0,
       marginLeft: 0,
       padding: 0,
-      '-webkit-appearance': 'none',
-      '-moz-appearance': 'none',
+      WebkitAppearance: 'none',
```

Comment:
Good to fix things like this when you're in the code - might be better to do this in another commit depending on what it is -- but this is probably small enough :smile: 

Curious if we still need noStyle here BTW -- last time we talked about the inputtime you where using the standard component